public/app/features/explore/utils/decorators.ts

```diff
@@ -34,8 +35,13 @@ export const decorateWithFrameTypeMetadata = (data: PanelData): ExplorePanelData
   const traceFrames: DataFrame[] = [];
   const nodeGraphFrames: DataFrame[] = [];
   const flameGraphFrames: DataFrame[] = [];
+  const customFrames: DataFrame[] = [];
 
   for (const frame of data.series) {
+    if (frame.meta?.preferredVisualisationPluginId && canFindPanel(frame)) {
```

Comment:
Is the check for preferredVisualizationPluginId needed here? It seems `canFindPanel` checks for it anyway:

```suggestion
    if (canFindPanel(frame)) {
```

Reply:
you are correct, I have removed the additional check.